addons/edi_sale/models/edi_sale_request_document.py

```diff
@@ -175,17 +174,6 @@ def report_invalid_records(self, doc):
         lines = SaleLineRequestRecord.search(error_domain)
         orders = SaleRequestRecord.search(error_domain)
         partners = PartnerRecord.search(error_domain)
-        message = []
-        if lines:
```

Comment:
Are these methods to build error messages on different models not required anymore?

Reply:
Yes we were discussing about these messages, now they are handled in a generic version. No need of these anymore, thats what was told in the google chat